docs/structure-of-a-contract.rst

```diff
@@ -15,22 +15,22 @@ Versions
 
 Vyper supports version pragma which is used to reject being compiled with future compiler versions that might introduce incompatible changes.
 
-::
+.. code-block:: python
+
+    # @version 0.1.0b13
 
-  # @version 0.1.0b13
-  
 The version pragma checks that the compiler version is not a major version.
 
 .. _structure-state-variables:
 
 State Variables
 ===============
 
-State variables are values which are permanently stored in contract storage.
+State variables are values which are permanently stored in contract storage. They must be initialized at deployment (outside of any functions). State variables are accessed via the :ref:`self<constants-self>` variable.
```

Comment:
"they must be initialized at deployment... Otherwise they will take on their corresponding zero value for their type"